src/crimson/os/seastore/segment_manager/block.cc

```diff
@@ -130,29 +130,94 @@ block_sm_superblock_t make_superblock(
   };
 }
 
+using check_create_device_ertr = BlockSegmentManager::access_ertr;
+using check_create_device_ret = check_create_device_ertr::future<>;
+static check_create_device_ret check_create_device(
+  const std::string &path,
+  size_t size)
+{
+  logger().error(
+    "block.cc:check_create_device path {}, size {}",
+    path,
+    size);
```

Comment:
why not use logger macro in this file?

Reply:
Mostly because I haven't converted it yet.  Also, block.cc doesn't use a Transaction, so the main purpose of the new macros isn't really relevant here.